delegates.rb

```diff
@@ -83,93 +84,79 @@ def redirect(options = {})
   def authorize(options = {})
     logger = Java::edu.illinois.library.cantaloupe.script.Logger
    
-    u_file_access = ['10.128.99.55','10.128.1.167','10.224.6.10','10.128.99.167','10.128.98.50','10.224.6.26','10.224.6.35','172.16.1.94', '66.234.38.35']
+    g_file_access = ['10.128.99.55','10.128.1.167','10.224.6.10','10.128.99.167','10.128.98.50','10.224.6.26','10.224.6.35','172.16.1.94', '66.234.38.35']

```

Comment:
might want to change this to full_res_file_access -- g files have same resolution as jp2, possibly s. In any case, we should probably treat them the same.